mobile/src/lib/filesystem.js

```diff
@@ -0,0 +1,85 @@
+const ERROR_GET_DIRECTORY = 'Error to get directory'
+const ERROR_WRITE_FILE = 'Error to write file'
+const ERROR_GET_FILE = 'Error to get file'
+const ERROR_GET_COZY_ENTRY = 'Error to get cozy entry'
+
+export const getRootPath = () => {
+  return window.cordova.platformId === 'android'
+    ? window.cordova.file.externalRootDirectory
+    : window.cordova.file.dataDirectory
+}
+
+export const getCozyPath = () => {
+  return 'Cozy/Cozy Files/'
+}
+
+export const getEntry = (path) => {
+  return new Promise((resolve, reject) => {
+    window.resolveLocalFileSystemURL(path, resolve, reject)
+  })
+}
+
+export const getCozyEntry = () => {
+  return new Promise((resolve, reject) => {
+    getEntry(getRootPath() + getCozyPath())
+    .then(resolve)
+    .catch(
+      getEntry(getRootPath())
+      .then(entry => {
+        getDirectory(entry, 'Cozy').then(entry => {
+          getDirectory(entry, 'Cozy Files').then(resolve).catch(reject)
+        }).catch(reject)
+      }).catch(reject)
+    )
+  })
+}
+
+export const getDirectory = (rootDirEntry, folderName) => {
+  return new Promise((resolve, reject) => {
+    rootDirEntry.getDirectory(folderName, { create: true }, resolve, error => {
+      console.log(ERROR_GET_DIRECTORY, folderName)
+      console.log(error)
+      reject(ERROR_GET_DIRECTORY)
+    })
+  })
+}
+
+export const writeFile = (fileEntry, dataObj) => {
+  return new Promise((resolve, reject) => {
```

Comment:
same comment: arrow function's first instruction's result is return by default (without `return` keyword).